spec/policies/topic_request_policy_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe TopicRequestPolicy do
+  let(:topic_request) { FactoryGirl.build(:topic_request) }
+  context 'the person is not signed in' do
+    subject { described_class.new(nil_person, topic_request) }
+    let(:nil_person) { nil }
+
+    describe '#create?' do
```

Comment:
I think this spec file is a little backwards. Normally you'd see something like:

```ruby
describe "#my_method" do
  subject { my_method }

  it "does something" do
  end

  context "context one" do
    it "does another thing" do
    end
  end

  context "context two" do
    it "does something else" do
    end
  end
end
```